bin/check_exercises.sh

```diff
@@ -33,39 +28,39 @@ return_code=0
 # An exercise worth testing is defined here as any top level directory with
 # a 'tests' directory and a .meta/config.json.
 for exercise in $files; do
-   exercise="$exercise/$target_dir"
+    exercise="$exercise/$target_dir"
```

Comment:
The formatting changes seem to be unrelated to the thing the PR is fixing. I'm happy to approve, but would recommend keeping unrelated in separate PRs in the future. It makes reviewing easier.

Reply:
alright, will do!